timemachine/parallel/utils.py

```diff
@@ -43,3 +43,7 @@ def batch_list(values: List, num_workers: int) -> List[List]:
     for i, value in enumerate(values):
         batched_values[i % num_workers].append(value)
     return list(batched_values.values())
+
+
+def flatten_list(results: List[List[Any]]) -> List[Any]:
+    return [i for j in results for i in j]
```

Comment:
Trying to get a better sense for whether this belongs in parallel/utils.py or if it's specific to a structure that exists in examples/run_smc_on_freesolv.py ... 

Would we ever need to flatten a structure that's nested more than one level deep, or has a mix of lists, tuples, dicts, etc?

Reply:
inlined in run_smc